content/en/metrics/custom_metrics/historical_metrics.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Historical Metrics Ingestion
+kind: documentation
+further_reading:
+- link: "/developers/dogstatsd/"
+  tag: "Documentation"
+  text: "Learn more about DogStatsD"
+- link: "/developers/community/libraries/"
+  tag: "Documentation"
+  text: "Official and Community created API and DogStatsD client libraries"
+---
+
+## Overview
+
+Historical Metrics Ingestion allows you to collect metric points with outdated timestamps. Timestamps that are older than one hour from the time of submission, but no older than your total metric retention period (which defaults to 15 months for all metrics).
+
+With Historical Metrics Ingestion, you can monitor a variety of new use cases with your metrics, such as outage recovery, overwriting invalid metrics, and managing IoT delays.
+
+## What is Historical Metric Ingestion?
+
+{{< img src="metrics/custom_metrics/historical_metrics/diagram_historical-metrics-ingestion_intro.png" alt="Diagram showing the ingestion flow for Historical Metrics" >}}
+
+Datadog classifies *historical metrics* as metric points with timestamps that are older than an hour relative to the time of submission. 
+
+For example, you emit a metric point at 1:00 PM EST, and the timestamp on that point reads 10:00 AM EST. This metric point is classified as a historical metric because it is delayed by 3 hours relative to the time of submission.
```

Comment:
```suggestion
For example, your metric (`exampleMetricA`) emits a value to Datadog at 1:00 PM EST, and the timestamp on that value is 10:00 AM EST. This metric value is classified as a _historical_ because it has a timestamp 3 hours older relative to the time of submission.
```